lib/keeper/controller.rb

```diff
@@ -7,22 +7,36 @@ def self.included(klass)
     end
 
     module InstanceMethods
+      JWT_REGEX = /[A-Za-z0-9\-_=]+\.[A-Za-z0-9\-_=]+\.?[A-Za-z0-9\-_=]+/
```

Comment:
The entire header, as defined by the oauth2 rfc, includes the `Bearer` prefix, this is to select around that, while not inherently making an assumption about other possible content.